examples_tests/utils.go

```diff
@@ -0,0 +1,53 @@
+package examples
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/aiven/aiven-go-client"
+	"github.com/gruntwork-io/terratest/modules/random"
+)
+
+// randName Returns randomized name for a given format specifier randName("foo-%s")
+func randName(format string) string {
```

Comment:
Not sure if we really need this function, we can perhaps move it's content to the `randNameGen`

Reply:
That's a handy one for small tests with a single name, so you don't have to make a generator just for a one var.